### ISSUE TITLE
Not bidirectional streaming note

### ISSUE THREAD

**Issue by eufat** [MelleDijkstra/python-grpc-chat#1]
If it a bidirectional streaming system, the proto file should be like this:
```
service ChatServer {
    // This bi-directional stream makes it possible to send and receive Notes between 2 persons
    rpc ChatStream (stream Note) returns (stream Note);
}
```

**Comment by melledijkstra** [comment]
@eufat You are probably right, although I might have had issues doing it this way (don't remember though, been a while). Will look into it when I have time.

**Comment by entrehuihui** [comment]
run python -m grpc_tools.protoc -I=proto/ --python_out=proto/ --grpc_python_out=proto/ proto/chat.proto  generate import chat_pb2 as chat__pb2. not  from . import chat_pb2 as chat__pb2

**Comment by jai-dewani** [comment]
Yes, I am also having issues working with truly bidirectional streams with gRPC in python.   
Especially having problem running two separate async loops for getting messages and sending messages in a single method
